praktikumsplaner-frontend/frontend/src/api/PraktikumsstellenService.ts

```diff
@@ -11,106 +11,78 @@ export default {
         return fetch(
             `${API_BASE}${PRAKTIKUMSSTELLE_BASE}/studium`,
             FetchUtils.getPOSTConfig(praktikumsstelle)
-        )
-            .then((response) => {
+        ).then((response) => {
+            FetchUtils.defaultResponseHandler(response);
+            if (response.ok) {
                 useSnackbarStore().showMessage({
-                    message: "☑ Speichern erfolgreich.",
+                    message: "☑ Praktikumsstelle erfolgreich angelegt",
```

Comment:
I think "... erfolgreich gemeldet" would be more comprehensive, as the end user does not necessarily know that "angelegt" means the "Ausbildungsleitung" can see it. He could think it was created but he still has to submit it and does not see any possibility to do that